### DISCUSSION TITLE
The webpage displays Stars 476, and the mobile terminal displays Stars 472

### DISCUSSION THREAD

**Discussion by 743859910** [community/community#27672] [Other Feature Feedback, Questions, & Ideas]
![Screenshot_2022-07-30-09-31-38-807_com github and](https://user-images.githubusercontent.com/84552324/181865319-dc274dc6-9983-4a8e-946e-4acad586d296.jpg)

![QQ截图20220730093311](https://user-images.githubusercontent.com/84552324/181865326-ad4b96fc-0ad4-4c30-91fe-2acf41714074.png)

**Comment by jfullstackdev** [comment]
but we cannot see it anymore right now, it's already private 

![image](https://user-images.githubusercontent.com/47092464/181866189-7d7cf7b1-0dd4-42cc-882e-e0d762ea3521.png)

it's just maybe some caching data

**Reply by 743859910** [reply]
Turn off private and still show 476 and 472 in browser incognito mode, it should not be cached, it has been like this for a long time
